lib/javascript/ajax.js

```diff
@@ -17,18 +17,18 @@
 
 // Some cutesy flashing thing while we run
 $(document).ajaxSend(function () {
-    $('#ajax-loading').show();
+    $('#ajax-loading').removeClass("hidden");
 });
 $(document).ajaxComplete(function () {
-    $('#ajax-loading').hide();
+    $('#ajax-loading').addClass("hidden");
```

Comment:
Any reason to do this? .hide() and .show() accomplish the same exact thing without needing a CSS deceleration.

Reply:
My idea was that all aspects of the web interface could be controlled through the theme. Styles applied with JavaScript are done directly to the element, making it hard to override, and change the behavior.

Then, if there are multiple elements that are being shown/hidden you can also apply the same styles to them, without adding additional selectors to a CSS rule.

Also, thinking about it, it might make more sense to toggle an active/inactive class instead.